polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -344,7 +346,7 @@ pub mod rococo {
 			},
 			// Ethereum Bridge
 			ethereum_inbound_queue: bridge_hub_rococo_runtime::EthereumInboundQueueConfig {
-				allowlist: Default::default(),
+				allowlist: vec![sp_core::H160([0x87, 0xd1, 0xf7, 0xfd, 0xfE, 0xe7, 0xf6, 0x51, 0xFa, 0xBc, 0x8b, 0xFC, 0xB6, 0xE0, 0x86, 0xC2, 0x78, 0xb7, 0x7A, 0x7d])],
```

Comment:
We should be setting the allowlist via a governance call. Can you add a ticket to track this?

Reply:
Resolved by https://linear.app/snowfork/issue/SNO-506.